scripts/github_issue_template.tmpl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 NXP Corp.
+# SPDX-License-Identifier: Apache-2.0
+'''
+    git hub issue template
+'''
+
+GITHUB_ISSUE_TITLE_TEMPLETE = '''
+tests-ci :${CASE_NAME} : test ${RESULT}
```

Comment:
tests-ci: ${CASE_NAME}: test ${RESULT}

->":" where incorrectly aligned.
In addition it will be clearer if you split the ${CASE_NAME} by "." and use only the first 2 names (module, submodule) with extra ":". Like: 

`tests-ci: ${MODULE}: ${SUBMODULE} test ${RESULT}`

Reply:
looks better, update